src/algebraic_geometry/elliptic_curve/weierstrass.lean

```diff
@@ -51,8 +51,8 @@ splitting field of `R` are precisely the $X$-coordinates of the non-zero 2-torsi
     if its discriminant is non-zero.
  * `weierstrass_curve.coordinate_ring.is_domain`: the coordinate ring of a Weierstrass curve is
     an integral domain.
- * `weierstrass_curve.degree_norm_smul_basis`: the degree of the norm of an element in the
-    coordinate ring as an `R[X]`-algebra in terms of the power basis.
+ * `weierstrass_curve.coordinate_ring.degree_norm_smul_basis`: the degree of the norm of an element
+    in the coordinate ring in terms of the power basis.
  * `elliptic_curve.nonsingular`: an elliptic curve is nonsingular at every point.
```

Comment:
Can you add `elliptic_curve.point_mk [nontrivial R] {x y : R} (h : E.equation x y) : E.point` as well?